alacritty_terminal/src/term/mod.rs

```diff
@@ -1916,8 +1896,8 @@ impl<T: EventListener> Handler for Term<T> {
         self.grid.reset(&Cell::default());
         self.alt_grid.reset(&Cell::default());
         self.scroll_region = Line(0)..self.grid.num_lines();
-        self.title = DEFAULT_NAME.to_string();
         self.title_stack.clear();
```

Comment:
Should we drop underlying buffer? I feel like we should release as much memory as we can on `reset`.